conans/client/remote_registry.py

```diff
@@ -98,8 +98,8 @@ def remote(self, name):
             try:
                 return Remote(name, remotes[name][0], remotes[name][1])
             except KeyError:
-                raise ConanException("No remote '%s' defined in remotes in file %s"
-                                     % (name, self._filename))
+                raise NoRemoteAvailable("No remote '%s' defined in remotes in file %s"
+                                        % (name, self._filename))
```

Comment:
leading whitespace?

Reply:
Nop, just aligning text for pep8, isn't?

Reply:
yes a bit misleading view in Github, sorry